configure

```diff
@@ -1062,6 +1067,28 @@ def configure_static(o):
         o['libraries'] += ['-static-libasan']
 
 
+def configure_gold(o):
+  try:
+    proc = subprocess.Popen(shlex.split('ld.gold') + ['-v'], stdin=subprocess.PIPE, 
+                            stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+    match = re.match(r"(GNU gold) .* ([0-9]\.[0-9]+)",
+                         proc.communicate()[0])
+    if match:
+      gold_version = match.group(2)
+      if gold_version > '1.1':
+        o['variables']['gold_linker'] = 'true'
```

Comment:
Don't use `o['variables']` if the settings aren't used in the *.gyp files.  Just use normal variables.